backend/doc/cli.md

```diff
@@ -59,17 +59,37 @@ optional arguments:
   --lang LANG           set user lang (ISO 639 format)
   -p PASSWORD, --password PASSWORD
                         set user password
-  --profile PROFILE     set user profile
+  --profile PROFILE     set user profile, valid values: users, trusted-users,
+                        administrators
   --timezone TIMEZONE   set user timezone
-  --send-email          Send mail to user
+  --send-email          send mail to created user about user account creation
+                        (email parameter required an email notification need
+                        to be activated)
 ```
 
-Example:
+Example creating user with email an defined password:
+
+```
+  tracimcli user create -e "john.does@example.tld" -p "superpassword" --send-email
+```
+
+Another example with username-only user (need email.required set to false) and no password setted:
 
 ```
-  tracimcli user create -e "john@john@john.john" -p "superpassword"
+  tracimcli user create -u "john" --profile "trusted-users" --lang "fr" --public-name "John Doe"
 ```
 
+:warning: if you don't provide password and don't use --send-email, no password will be created. you can use tracimcli user update to set the password
+after user creation.
+
+:warning: if you don't provide password and use --send-email, an autogenerated password will be created and send to user by email, example:
+
+```
+  tracimcli user create -e "john.does@example.tld" --send-email --lang "fr"
+```
```

Comment:
this example need to be show just after example with password and parameter --send-email.